Source/JavaScriptCore/wasm/WasmStreamingPlan.cpp

```diff
@@ -42,7 +42,7 @@ namespace WasmStreamingPlanInternal {
 static constexpr bool verbose = false;
 }
 
-StreamingPlan::StreamingPlan(VM& vm, Ref<ModuleInformation>&& info, Ref<LLIntPlan>&& plan, uint32_t functionIndex, CompletionTask&& task)
+StreamingPlan::StreamingPlan(VM& vm, Ref<ModuleInformation>&& info, Ref<LLIntPlan>&& plan, FunctionCodeIndex functionIndex, CompletionTask&& task)
     : Base(vm, WTFMove(info), WTFMove(task))
     , m_plan(WTFMove(plan))
     , m_functionIndex(functionIndex)
```

Comment:
I think you missed `m_functionIndex` of `StreamingPlan`. It should be `FunctionCodeIndex`.

Reply:
I think it's there?

Reply:
Oops, that's right.